content/websockets/adapter.md

```diff
@@ -44,20 +44,24 @@ import { createAdapter } from '@socket.io/redis-adapter';
 import { createClient } from 'redis';
 
 export class RedisIoAdapter extends IoAdapter {
-  private adapterConstructor: ReturnType<typeof createAdapter>;
+  private adapterConstructor?: ReturnType<typeof createAdapter>;
 
   async connectToRedis(): Promise<void> {
-    const pubClient = createClient({ url: `redis://localhost:6379` });
+    const pubClient = createClient({
+      url: process.env.REDIS_URL || 'redis://localhost:6379',
```

Comment:
I don't think that we should assume tha there is a env var for this as we will make this code snippet too opinative. We can leave that to the user instead